features/abortsignal-any.yml.dist

```diff
@@ -12,3 +12,6 @@ status:
     firefox_android: "124"
     safari: "17.4"
     safari_ios: "17.4"
+compat_features:
+  # Same status as overall feature
```

Comment:
A nice-to-have thing would be to omit this when there's only one group of keys (e.g., every single single-entry feature, but also features where all of the keys landed at the same time).